client/src/api.ts

```diff
@@ -264,29 +317,48 @@ export function setSigner(id: number, gasPrice: string): Promise<any> {
   return secureFetchNoResponse(`${API_BASE}/signers/${id}`, {
     method: 'PUT',
     headers: { 'Content-Type': 'application/json' },
-    body: JSON.stringify({gas_price: gasPrice})
+    body: JSON.stringify({ gas_price: gasPrice }),
   });
 }
 
-export function getTransactions(limit: number, offset: number, status: string): Promise<PaginatedTransactions> {
+export function getNotifications(
+  limit?: number,
+  offset?: number,
+  type?: string,
+  event_id?: number
+): Promise<PaginatedNotifications> {
+  const params = queryString.stringify({ limit, offset, type, event_id });
+  return secureFetch(`${API_BASE}/notifications?${params}`);
+}
+
+export function getTransactions(
+  limit: number,
+  offset: number,
+  status: string
+): Promise<PaginatedTransactions> {
+  // TODO: use queryString library to avoid sending empty params
```

Comment:
Hey @ragonzal we've left this comment in order to start using `query-string` library and avoid sending empty or null parameters to the backend. I.e line [339](https://github.com/Xivis/poap/pull/37/files#diff-2d9f3bc345812a65da64d03b7d5b8dcdR330)